src/core/openthread.cpp

```diff
@@ -991,6 +994,25 @@ void HandleActiveScanResult(void *aContext, Mac::Frame *aFrame)
     return;
 }
 
+ThreadError otEnergyScan(uint32_t aScanChannels, uint16_t aScanDuration, otHandleEnergyScanResult aCallback,
```

Comment:
@luwannest this is intended as the API ot exposes to outside (say through spinel/wpantund) to issue energy-scan. I think the tlvs should be handled internally or maybe through other related  apis.